### ISSUE TITLE
Hover states not included after purge

### ISSUE THREAD

**Issue by andersklenke** [rails/tailwindcss-rails#15]
Hover states works in development but not in production.

I use these hover states for a button `hover:bg-green-600 hover:text-white hover:shadow-lg` but when css is purged, these classes are not included. 

In the `tailwind.css` file this is what `hover:bg-green-600` looks like, so maybe it's due to structure of this, that the Purger doesn't work? 
```css
.hover\:bg-green-600:hover {
  --tw-bg-opacity: 1;
  background-color: rgba(5, 150, 105, var(--tw-bg-opacity));
}
```

**Comment by dhh** [comment]
Fixed in 0.2.1 ✌️❤️

**Comment by andersklenke** [comment]
@dhh Thanks for this! This introduces an issue where group-hover classes are not removed, eg: `group-hover:text-gray-900` is not purged. Unfortunately my regex skills are not strong enough to fix it.
